examples/redis_plugin/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"time"
+
+	"github.com/ligato/cn-infra/core"
+	"github.com/ligato/cn-infra/datasync/kvdbsync"
+	"github.com/ligato/cn-infra/datasync/resync"
+	"github.com/ligato/cn-infra/db/keyval"
+	"github.com/ligato/cn-infra/db/keyval/redis"
+	"github.com/ligato/cn-infra/flavors/local"
+	log "github.com/ligato/cn-infra/logging/logroot"
+	"github.com/namsral/flag"
+)
+
+// Main allows running Example Plugin as a statically linked binary with Agent Core Plugins. Close channel and plugins
+// required for the example are initialized. Agent is instantiated with generic plugin (Status check, and Log)
+// and example plugin which demonstrates use of Redis flavor.
+func main() {
+	// Init close channel used to stop the example
+	exampleFinished := make(chan struct{}, 1)
+
+	// Start Agent with ExampleFlavor (combination of ExamplePlugin & reused cn-infra plugins)
+	flavor := ExampleFlavor{closeChan: &exampleFinished}
+	agent := core.NewAgent(log.StandardLogger(), 15*time.Second, append(flavor.Plugins())...)
+	core.EventLoopWithInterrupt(agent, exampleFinished)
+}
+
+// Redis flag to load config
+func init() {
+	flag.String("redis-config", "redis.conf",
+		"Location of the redis configuration file")
+}
+
+// ExampleFlavor is a set of plugins required for the redis example.
+type ExampleFlavor struct {
```

Comment:
could you please move the example flavor & Deps struct to dedicated file inside example dir (Milan is currently doing that for all other examles)? thanks

Reply:
done